src/components/views/settings/tabs/RolesRoomSettingsTab.js

```diff
@@ -116,7 +116,8 @@ export default class RolesRoomSettingsTab extends React.Component {
     _onPowerLevelsChanged = (value, powerLevelKey) => {
         const client = MatrixClientPeg.get();
         const room = client.getRoom(this.props.roomId);
-        let plContent = room.currentState.getStateEvents('m.room.power_levels', '').getContent() || {};
+        const plEvent = room.currentState.getStateEvents('m.room.power_levels', '');
```

Comment:
Should the JS SDK instead return a dummy object so that the methods still work without testing for them everywhere? This seems like it will get quite annoying to remember to do this everywhere.

Reply:
Probably not as that could easily break things that aren't Riot.

Reply:
Okay, seems fine to go with this for now at least.